Resources/Prototypes/_Goobstation/Research/arsenal.yml

```
@@ -37,6 +37,21 @@
   technologyPrerequisites:
   - Ripley2
 
+- type: technology
+  id: AdvancedWeaponry
+  name: research-technology-advancedweaponry
+  icon:
+    sprite: /Textures/_EinsteinEngines/Objects/Weapons/Melee/terminus.rsi
+    state: icon
+  discipline: Arsenal
+  tier: 2
+  cost: 13500
```

Same reasoning as above, this is a weapon for only one specie, and noone researched x-ray cannons anyways since they suck against nukies (which is the only reason you'd want to research military science in the first place).
```suggestion
  cost: 10000
```